tf.yaml

```diff
@@ -3,5 +3,6 @@ aws_instance:
     - Name
     - terraform
 
-  volume_tags:
+  lifecycle:
+    - create_before_destroy
```

Comment:
Cool, can we add some more stuff to this? Like the wait_for_user_data thing and maybe the public_ip_address = false thing?

Reply:
This check can't verify values atm, it will take a long time to implement this feature.

Reply:
Or maybe we can just add that extra stuff to our configuration file. Sorry I forgot which repo I was in :)